### PR TITLE
Add test for utils/filters

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
Nice, coverage: `81.2%` of statements in `filters.go`

File: pkg/utils/filters_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package utils
```
Comment (chengxiangdong) [inline pkg/utils/filters_test.go:1]: License required to apply to this file

Comment (niuzhenguo) [reply pkg/utils/filters_test.go:17]: done

**Pull request merged**